staging/src/k8s.io/client-go/tools/cache/shared_informer.go

```diff
@@ -524,6 +656,21 @@ func (s *sharedIndexInformer) AddEventHandlerWithResyncPeriod(handler ResourceEv
 	}
 }
 
+func (s *sharedIndexInformer) RemoveEventHandler(handler ResourceEventHandler) bool {
+	for i, listener := range s.processor.listeners {
+		// TODO: check if handlers can be compared, error out if not, document it.
+		if listener.handler == handler {
+			s.processor.removeListener(i)
+			return true
+		}
+	}
+	return false
+}
+
+func (s *sharedIndexInformer) EventHandlerCount() int {
+	return len(s.processor.listeners)
```

Comment:
Similar to above. Need a lock and maybe we shouldn't reach into the internals of the processor directly.

Reply:
Done